src/lf_stack.ml

```diff
@@ -0,0 +1,62 @@
+(** Trieber's Lock Free stack *)
+
+type 'a node =
+  | Nil
+  | Next of 'a * 'a node Atomic.t
+
+type 'a t =
+  { head : 'a node Atomic.t ;
+    tail : 'a node Atomic.t }
```

Comment:
tail is not used, perhaps a leftover from this being implemented on the base of msqueue?